content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
 
 The pod annotation for Kiali is: `kiali.io/runtimes: go`
 
+
+=== Envoy
+
+The Envoy sidecars that come with Istio are also supplying link:https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats[some internal metrics], that you can see in Kiali. They are not the same than the metrics reported by Istio Telemetry, and of which Kiali makes an intensive usage. Some of them can be redundant.
+
+Unlike the other Runtimes dashboards, this one has to be enabled explicitely with the pod annotation `kiali.io/runtimes: envoy` (there's no automatic discovery configured).
```

Comment:
Required change:
`explicitely -> explicitly`
Optional change:
`"has to be enabled explicitely with the pod" -> "needs to be explicitly enabled with pod"`

Reply:
Unlike the other Runtimes dashboards, there is no automatic discovery configured for Envoy.  You must explicitly enable the Envoy dashboard  with the pod annotation `kiali.io/runtimes: envoy`.